jscomp/super_errors/super_typecore.ml

```diff
@@ -253,6 +325,7 @@ let report_error env ppf err =
 
 (* This will be called in super_main. This is how you'd override the default error printer from the compiler & register new error_of_exn handlers *)
 let setup () =
+  Misc.Color.setup !Clflags.color;
```

Comment:
Should this be here? How come it's not needed in `typecore.ml`?

Reply:
I don't know, but without it the color argument isn't respected

Reply:
I think color is already triggered in bucklescript main entry.
note `bsc` is a wrapper around `bsc.exe`, so bsc.exe will give your color but bsc may not, but you can always turn it on by using `bsc -color always`
Can you check if this line is necessary?

Reply:
Yup it is -- I ran `bsc.exe -color never` and without this line, it would still output colors from this file.